lmfdb/abvar/fq/isog_class.py

```diff
@@ -231,6 +236,20 @@ def properties(self):
             ("Geometrically simple", "yes" if self.is_geometrically_simple else "no"),
             ("Primitive", "yes" if self.is_primitive else "no"),
         ]
+        # Cyclicity information (new)
+        ic = getattr(self, "is_cyclic", None)
+        if ic is True:
+            cyclic_val = "yes"
+        elif ic is False:
+            cyclic_val = "no"
+        else:
+            cyclic_val = "unknown"
+        props.append(("Cyclic group of points", cyclic_val))
+
+        primes = getattr(self, "noncyclic_primes", None)
```

Comment:
Same comments as above: you can use `self.noncyclic_primes` and this should go in the invariants section not properties box.